build.sbt

```diff
@@ -32,7 +32,7 @@ ThisBuild / githubWorkflowBuildMatrixInclusions ++=
   }
 
 ThisBuild / githubWorkflowBuildMatrixInclusions ++=
-  crossScalaVersions.value.filter(_.startsWith("2.")) map { scala =>
+  crossScalaVersions.value.filterNot(_ == Scala30) map { scala =>
```

Comment:
- https://repo1.maven.org/maven2/org/scala-native/nscplugin_3.1.0/
- https://scala-native.readthedocs.io/en/latest/changelog/0.4.3-RC1.html

> The minimal version of Scala 3 supported by Scala Native is 3.1.0, due to fatal blockers in Scala 3.0.x.

Reply:
If it were up to me, I would release Scala 3 native with a minor version increase (1.16.0), which would only be published for Scala 3.1.